packages/renderer/src/lib/volume/VolumeColumnName.svelte

```diff
@@ -0,0 +1,16 @@
+<script lang="ts">
```

Comment:
Here, we could have a ClickableStringColumn? Passing the url and string value

Reply:
Yes, but again I think this is a feature we could add later. We have a bug for the fact that you can only click on one column today, we need to decide how you get to details vs other table actions.